wordpress-mobile/WordPress-Android
# Intercept wp.me post shortlinks

Adds support for intercepting `wp.me` shortlinks to posts and opening them directly in the Reader.

To test:
* Try to open this shortlink http://wp.me/pf2B5-9EE by opening this PR in Chrome or by sharing the link through some other app you can open (e.g. Slack)
* The app should come up and display the post (https://en.blog.wordpress.com/2016/09/26/the-best-of-wordpress-september-2016/)
